distribution/build.gradle

```diff
@@ -481,7 +481,7 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
         }
       }
     }
-    
+
```

Comment:
Accidental?

Reply:
Looks like this is coming from `./gradlew spotlessApply ` when I ran on local.